nx/lib/nx.ex

```diff
@@ -6133,7 +6133,7 @@ defmodule Nx do
       #Nx.Tensor<
         f64[2][2]
         [
-          [-3.141592653589793, -0.0],
+          [-3.141592653589793, 0.0],
```

Comment:
Is this change correct?

Reply:
Yes, it was intentional, for consistency with the custom handling of -0.0 in complex_to_string